src/ImageReconstruction.jl

```diff
@@ -1,32 +1,22 @@
 module ImageReconstruction
 
+using Base.Threads
+using Interpolations
 using FFTW
 
 export radon, iradon
 
 """
-radon transform
+    radon(I::AbstractMatrix, θ::AbstractRange, t::AbstractRange)
 
-https://en.wikipedia.org/wiki/Radon_transform
+Radon transform of a image `I` producing a sinogram from view angles `θ` in
+radians and detector sampling `t`.
 """
-function radon end
+function radon(I::AbstractMatrix, θ::AbstractRange, t::AbstractRange)
+    P = zeros(eltype(I), length(t), length(θ))
+    nr, nc = size(I)
 
-"""
-inverse radon transform
-
-https://en.wikipedia.org/wiki/Radon_transform
-"""
-function iradon end
-
-
-"""
-Radon transform of an image using a pixel-driven algorithm.
-"""
-function radon(image::AbstractMatrix, θ::AbstractRange, t::AbstractRange)
-    P = zeros(eltype(image), length(t), length(θ))
-    nr, nc = size(image)
-
-    for i in 1:nr, j in 1:nc
+    for i = 1:nr, j = 1:nc
```

Comment:
```suggestion
    for j in ax2, i in ax1
```
Note that I reversed your loop order. It should be *much* more efficient that way. See https://julialang.org/blog/2013/09/fast-numeric/